metisp/README.md

```diff
@@ -22,7 +22,7 @@ Including the following two line in your .bashrc
 export METIS_SOFTPATH='<path_of_METIS_pipeline>' 
 export PYTHONPATH=$METIS_SOFTPATH
 export SOF_DATA='<path_to_data>'
-export PYCPL_RECIPE_DIR='<path_of_METIS_pipeline>/prototypes/recipes/'
+export PYCPL_RECIPE_DIR='<path_of_METIS_pipeline>/metisp/pymetis/src/pymetis/recipes/'
```

Comment:
```suggestion
export PYCPL_RECIPE_DIR='<path_of_METIS_pipeline>/metisp/pyrecipes/'
```
pyrecipes is the official path that should be used. So please make sure all recipes are actually imported there